posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+---
+title: "Getting Started with the `napari-cosmx` plugin"
+author:
+  - name: Evelyn Metzger
+    affiliations: 
+      - ref: nstg
+      - ref: eveilyeverafter
+affiliations:
+  - id: nstg
+    name: NanoString Technologies, Inc.
+    city: Seattle
+    state: WA
+  - id: eveilyeverafter
+    name: "Github: [@eveilyeverafter](https://github.com/eveilyeverafter)"
+toc: true
+toc-title: Contents
+toc-depth: 3
+toc-expand: 2
+toc-location: left
+number-sections: true
+number-depth: 4
+date: "2024-04-19"
+categories: [visualization, napari]
+draft: false
+image: figures/prostate.gif
+---
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-pancreas"
+#| fig-cap: "Nine Fields of View (FOVs) of the whole-transcriptome Pancreas dataset generated with the `napari-cosmx` plugin. DAPI and PanCK is shown in blue and green, respectively. Endocrine cells in the Islets of Langerhans can be idenified by their transcript abundance (points) of marker genes. Red = _GCG_ (alpha cells), orange = _INS_ (beta cells)."
+
+knitr::include_graphics("./figures/pancreas.png")
+```
+
+# Introduction 
+
+The AtoMx Spatial Informatics Portal (SIP) is an excellent solution for analyzing 
+multi-omics spatial data generated on the CosMx Spatial Molecular Imager (SMI).
+It offers seamless integration of data analysis with data viewer in an intuitive 
```

Comment:
. . . with _a_ data viewer . . .

Reply:
done. d7986cbca65175d99063e11e2e6b476c0819d535
I also added TM symbols with first use of AtoMx and CosMx.